common/djangoapps/student/tests/test_create_account.py

```diff
@@ -945,7 +945,7 @@ def test_special_chars_with_feature_enabled(self):
         """
 
         invalid_params = self.url_params.copy()
-        invalid_params['username'] = '**john**'
+        invalid_params['username'] = '\\/john**'
```

Comment:
Why would we want to _allow_ `**` in a username? 

```suggestion
        invalid_params['username'] = '**john**'
```

I have another suggestion which is to test on more than one test case:

```python
    @mock.patch.dict(settings.FEATURES, {'ENABLE_UNICODE_USERNAME': True})
    @ddt.data('**john', 'john**', '\\/john')
    def test_special_chars_with_feature_enabled(self, invalid_username):
        invalid_params['username'] = invalid_username
```